pkgs/applications/version-management/git-and-tools/git-machete/default.nix

```diff
@@ -1,26 +1,36 @@
-{ lib, buildPythonApplication, fetchPypi
-, installShellFiles, pbr
-, flake8, mock, pycodestyle, pylint, stestr, tox
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, installShellFiles
+, git
+, stestr
 , nix-update-script
-, testVersion, git-machete
+, testVersion
+, git-machete
 }:
 
 buildPythonApplication rec {
   pname = "git-machete";
   version = "3.3.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0mq6hmb3wvj0ash27h4zyl46l3fikpf0mv3ng330lcy6v7bhy5b8";
+  src = fetchFromGitHub {
+    owner = "virtuslab";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0sx45y1d1v6y66msjc1lw9jhjppgbxqj145kivmd7lr6ccw68kav";
   };
 
-  nativeBuildInputs = [ installShellFiles pbr ];
+  nativeBuildInputs = [ installShellFiles ];
 
-  checkInputs = [ flake8 mock pycodestyle pylint stestr tox ];
+  checkInputs = [ git stestr ];
+
+  postCheck = ''
+    stestr run
+  '';
 
   postInstall = ''
-      installShellCompletion --bash --name git-machete completion/git-machete.completion.bash
-      installShellCompletion --zsh --name _git-machete completion/git-machete.completion.zsh
+    installShellCompletion --bash --name git-machete completion/git-machete.completion.bash
+    installShellCompletion --zsh --name _git-machete completion/git-machete.completion.zsh
```

Comment:
There's an `_` in git-machete, is this expected?

Reply:
pretty usre it is intended, usually the zsh completion files have a `_` in the front


also see [nixpkgs doc](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/build-support/setup-hooks/install-shell-files.sh#L88)
> `installShellCompletion --zsh --name _foobar share/completions.zsh`